examples/example_util.py

```diff
@@ -53,13 +53,18 @@ def show_metrics(rbm, performance):
 
 def compute_reconstructions(rbm, v_data, fit, n_recon=10, vertical=False):
     sampler = fit.DrivenSequentialMC(rbm)
-    data_state = State.from_visible(v_data, rbm)
+    data = v_data[0] if isinstance(v_data, tuple) else v_data
```

Comment:
the reason is line 63, where I want to know if the input contains also targets, if so, I want to pull out predictions and print them